webpack.config.component.js

```diff
@@ -54,6 +54,21 @@ const esConfig = {
 
 const umdConfig = {
   ...baseConifg,
+  externals: {
+    // Only because this is the library target.
+    react: {
+      commonjs: 'react',
+      commonjs2: 'react',
+      amd: 'react',
+      root: 'React'
+    },
+    'react-dom': {
+      commonjs: 'react-dom',
+      commonjs2: 'react-dom',
+      amd: 'react-dom',
+      root: 'ReactDOM'
+    }
+  },
```

Comment:
Are all these necessary?

```suggestion
  externals: {
    // Only because this is the library target.
    react: 'React',
    'react-dom': 'ReactDOM'
  },
```